specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/nova.json

```diff
@@ -0,0 +1,1058 @@
+                "type": "number",
+                "format": "double",
+                "description": "The percentage of the data owned by the node per datacenter times the replication factor (e.g., 33.3, or null if the data is not available). For example, a node can own 33% of the ring, but shows 100% if the replication factor is 3. For non-system keyspaces, the endpoint percentage ownership information is shown."
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "BackupResource": {
+      "description": "A restorable backup of a Cassandra cluster.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "properties": {
+            "timestamp": {
+              "type": "string",
+              "format": "date-time",
+              "description": "The time this backup was taken, formatted like 2021-01-21T17:35:21"
+            }
+          }
+        }
+      }
+    },
+    "ListBackups": {
+      "description": "List of restorable backups for a Cassandra cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of backups.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/BackupResource"
+          }
+        }
+      }
+    },
+    "ListDataCenters": {
+      "description": "List of managed Cassandra data centers and their properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of data centers.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/DataCenterResource"
+          }
+        }
+      }
+    },
+    "SeedNode": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "IP address of this seed node.",
+          "type": "string"
+        }
+      }
+    },
+    "Certificate": {
+      "type": "object",
+      "properties": {
+        "pem": {
+          "description": "PEM formatted public key.",
+          "type": "string"
+        }
+      }
+    },
+    "DataCenterResource": {
+      "description": "A managed Cassandra data center.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
```

Comment:
DataCenterResource is missing the provisioningState property

Reply:
Added.